flipperdevices/flipperzero-firmware
# Add File Naming setting for more detailed naming

# What's new

Solved Issue #2598 (Feature Request) by adding more detail to file names through the use of a non-invasive setting named 'File Naming'.

# Verification 

1. Go to: Settings > System > File Naming > Set to 'Detailed'
2. Create any file that is an internal app which supports the naming convention (Sub-Ghz, NFC, iButton, RFID)
3. Save the file

You can check the difference by setting the filename mode to 'Default'.

# Checklist (For Reviewer)
